app/src/main/kotlin/com/intive/picover/common/viewmodel/state/ViewModelState.kt

```diff
@@ -1,19 +0,0 @@
-package com.intive.picover.common.viewmodel.state
-
-sealed class ViewModelState<out T> {
```

Comment:
Instead of renaming it, which is displayed as removed file and added a new one `DeprecatedViewModelState.kt`, please use `@Deprecated` annotation. It allows you to specify the type to replaced with :)